_episodes/07-thresholding.md

```diff
@@ -126,87 +124,45 @@ Now is where the main work of the program takes place.
 
 ~~~
 # blur and grayscale before thresholding
-blur = cv2.cvtColor(src = image, code = cv2.COLOR_BGR2GRAY)
-blur = cv2.GaussianBlur(src = blur, 
-    ksize = (k, k), 
-    sigmaX = 0)
+blur = skimage.color.rgb2gray(image)
+blur = skimage.filters.gaussian(blur, sigma=k) 
 ~~~
 {: .python}
 
 First, we convert the 
-image to grayscale and then blur it, using the `cv2.GaussianBlur()` function we
-learned about in the [Blurring]({{ page.root }}/06-blurring/) episode. The
-image passed into the thresholding function must be grayscale. 
+image to grayscale and then blur it, using the `skimage.filter.gaussian()` function we
+learned about in the [Blurring]({{ page.root }}/06-blurring/) episode. We convert the input image to grayscale for easier thresholding. 
 
-The fixed-level thresholding is performed with the `cv2.threshold()` function 
-call. 
+The fixed-level thresholding is performed using numpy comparison operators.
 
 ~~~
 # perform inverse binary thresholding 
-(t, maskLayer) = cv2.threshold(src = blur, 
-    thresh = t, 
-    maxval = 255, 
-    type = cv2.THRESH_BINARY_INV)
+mask = blur < t
 ~~~
 {: .python}
 
-We pass in four parameters. The first, `blur`, is our blurred grayscale
-version of the image. Next is our threshold value `t`. The third parameter is
-the value to be used for pixels that are turned on during the thresholding, 255. 
-Finally, we pass in a constant telling the function what kind of 
-thresholding to apply, `cv2.THRESH_BINARY_INV`. This instructs the function to 
-turn on pixels with color values below the threshold value T and turn off
-pixels with color values above T. 
-
-The function returns a tuple of two items: the value used for `t` and a new, 
-two-dimensional NumPy array representing the binary image created by the 
-thresholding operation. You may be wondering why the function returns `t`, since 
-we passed the value in. There are other ways of using the `cv2.threshold()` 
-function in which the threshold value is calculated automatically. For now, just 
-focus on the binary image created by the function, `maskLayer`.
-
-Since `maskLayer` is a binary image, it has only one color channel, and each 
-value in the corresponding NumPy array is either 0 or 255. Here is a 
+Here, we want to turn "on" all pixels which have values smaller than the threshold, 
+so we use the `less operator <` to compare the blurred image `blur` to the threshold `t`. 
+The operator reurns a binary image, that we capture in the variable `mask`.
+It has only one channel, and each of its values is either 0 or 1. Here is a 
 visualization of the binary image created by the thresholding operation.
 The program used parameters of k = 7 and T = 210 to produce this image. You can
```

Comment:
you should probably use something like `sigma=1.0`, `truncate=3.5` for the Gaussian and ,`T=0.82` for the threshold.